core/pom.xml

```diff
@@ -100,7 +100,12 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
```

Comment:
@ewencp and I discussed this in person. I'll use the license generator tool to generate a `licenses/` directory that contains both notices and licenses. I'll also change `NOTICE` in the root directory to be the summary HTML file generated from the script. I won't do this in this pull request, because it will make tons of changes. I'll do this in a follow-on pull request.